qt_ui/windows/basemenu/airfield/QAircraftRecruitmentMenu.py

```diff
@@ -92,25 +92,20 @@ def buy(self, unit_type):
                     self, "No space for additional aircraft",
                     f"There is no parking space left at {self.cp.name} to accommodate another plane.", QMessageBox.Ok)
                 return
-
+                
         super().buy(unit_type)
         self.hangar_status.update_label()
 
     def sell(self, unit_type: UnitType):
         # Don't need to remove aircraft from the inventory if we're canceling
         # orders.
-        if self.pending_deliveries.units.get(unit_type, 0) <= 0:
-            global_inventory = self.game_model.game.aircraft_inventory
-            inventory = global_inventory.for_control_point(self.cp)
-            try:
-                inventory.remove_aircraft(unit_type, 1)
```

Comment:
This still needs to be handled. Without this, sold aircraft may still be fragged, and already fragged aircraft may be sold.

It's probably not clear. The `base` object tracks the total amount of vehicles (ground and air) that are at the CP. The inventory tracks aircraft that are available for mission planning. When an aircraft is sold it needs to be taken out of the inventory so it isn't used for the turn's missions.

Reply:
Fixed, and thanks for the explanation of the logic here.